app/views/layouts/application.html.erb

```diff
@@ -1,14 +1,83 @@
 <!DOCTYPE html>
-<html>
-<head>
-  <title>Lawyer</title>
-  <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
-  <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
-  <%= csrf_meta_tags %>
-</head>
-<body>
-
-<%= yield %>
-
-</body>
-</html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name='yandex-verification' content='491fa3000fe6a3c3'/>
+    <meta name="google-site-verification" content="N_nGPZaPiBcDucGlliHlhwRujBuQLhMYWKFhzlcoc54"/>
+    <meta name="description" content="Проведение юридических консультаций в бесплатно"/>
+    <title><%= content_for?(:title) ? yield(:title).capitalize + ' - Juristsovet' : "Бесплатная юридическая консультация - Juristsovet" %></title>
+
+    <%= stylesheet_link_tag "application" %>
+    <%#= javascript_include_tag "vendor/modernizr.min" %>
+    <%= csrf_meta_tags %>
+  </head>
+
+  <body>
+    <%= yield %>
+    <div id="myModal" title="Уведомление">
+      Благодарим за заявку. Скоро с вами свяжется адвокат!
+    </div>
+
+    <%= javascript_include_tag "application" %>
+    <script type="text/javascript"
+            src="http://api.leadiacloud.com/wi1.js?p=lawyer&w=5282&t=fabric&ft=fabricform&color=blue&f=roboform"></script>
+
+    <!-- Yandex.Metrika counter -->
+    <script type="text/javascript">
```

Comment:
it's not the best place to hold JS code. All code should be moved to js file. Even if we need to load a JS script a dedicated page it could be done cleaner, like:
in js file with name f.ex. mectrics.js 
```
function Metrics() {
  this.yandexMetrics = {.....};
  this.googleMetrics = {.....};
  this.leadiaCloud = {...};
}
``` 

in selected page could be added something like this to call existed function on page load:
```
<script type="text/javascript">
  metrics = new Metrics();
  metrics.yandexMetrics();
  ....
<script>
```